Source/WebCore/css/html.css

```diff
@@ -1033,12 +1029,6 @@ input:is([type="checkbox"], [type="radio"]):checked:disabled {
 }
 #endif
```

Comment:
Could be good adding a comment indicating the end of the #endif but I guess the rest of the file doesn't do it.

Like: https://searchfox.org/wubkat/search?q=%23endif+%2F%2F&path=&case=false&regexp=false

Reply:
I'll leave that for when I want to make more substantive changes.